src/core/memory.cc

```diff
@@ -1659,6 +1659,18 @@ static human_readable_value to_hr_number(uint64_t number) {
     return to_human_readable_value(number, 1000, 10000, suffixes);
 }
 
+}
+
+}
+
+#if FMT_VERSION >= 90000
+template <> struct fmt::formatter<struct seastar::memory::human_readable_value> : fmt::ostream_formatter {};
+#endif
```

Comment:
Yikes!!! Now I see why it was with forward declaration. Would you please return it back? Sorry for this back-and-forth, breaking the namespace for this glue code looks horrible.